web/public/templates/dojos/user-profile.dust

```diff
@@ -129,7 +129,7 @@
             <div class="dojo-container" ng-repeat="dojo in profile.dojos">
               <img ng-click="viewDojo(dojo)" src="/img/dojo-default-logo.png" class="dojo-thumbnail pointer">
```

Comment:
You can wrap this img in an <a> tag with ng-href instead of using ng-click on the img. This will then provide a consistent behaviour for both the image and link.